docs/topics/testing/tools.txt

```diff
@@ -1596,8 +1596,8 @@ your test suite.
       closed or the HTML document ends.
     * Empty tags are equivalent to their self-closing version.
     * The ordering of attributes of an HTML element is not significant.
-    * Attributes without an argument are equal to attributes that equal in
-      name and value (see the examples).
+    * `Boolean attributes`_ (like ``checked``, ``disabled``, ...) without an
+      argument are equal to attributes that equal in name and value (see the examples).
```

Comment:
I think we can chop the link:
```suggestion
    * Boolean attributes (like ``checked``) without an argument are equal to
      attributes that equal in name and value (see the examples).
```
Also, `versionchanged` annotation seems appropriate, e.g.
```
.. versionchanged:: 4.0

    In older versions, non-boolean attributes without an argument were equal to
    attributes that equal in name and value.
```

Reply:
I tweaked the `versionchanged` text a bit from what you suggested.